modules/cnf-collecting-low-latency-tuning-debugging-data-for-red-hat-support.adoc

```diff
@@ -72,26 +45,57 @@ In earlier versions of {product-title}, the Performance Addon Operator provided
 
 . Navigate to the directory where you want to store the `must-gather` data.
 
-. Run the `oc adm must-gather` command with one or more `--image` or `--image-stream` arguments. For example, the following command gathers both the default cluster data and information specific to the Node Tuning Operator:
+. Collect debugging information by running the following command:
 +
-[source,terminal,subs="attributes+"]
+[source,terminal]
 ----
-$ oc adm must-gather \
- --image-stream=openshift/must-gather \ <1>
-
- --image=registry.redhat.io/openshift4/performance-addon-operator-must-gather-rhel8:v{product-version} <2>
+$ oc adm must-gather
+----
++
+.Example output
++
+[source,terminal]
+----
+[must-gather      ] OUT Using must-gather plug-in image: quay.io/openshift-release
+When opening a support case, bugzilla, or issue please include the following summary data along with any other requested information:
+ClusterID: 829er0fa-1ad8-4e59-a46e-2644921b7eb6
+ClusterVersion: Stable at "<cluster_version>"
+ClusterOperators:
+	All healthy and stable
+
+
+[must-gather      ] OUT namespace/openshift-must-gather-8fh4x created
+[must-gather      ] OUT clusterrolebinding.rbac.authorization.k8s.io/must-gather-rhlgc created
+[must-gather-5564g] POD 2023-07-17T10:17:37.610340849Z Gathering data for ns/openshift-cluster-version...
+[must-gather-5564g] POD 2023-07-17T10:17:38.786591298Z Gathering data for ns/default...
+[must-gather-5564g] POD 2023-07-17T10:17:39.117418660Z Gathering data for ns/openshift...
+[must-gather-5564g] POD 2023-07-17T10:17:39.447592859Z Gathering data for ns/kube-system...
+[must-gather-5564g] POD 2023-07-17T10:17:39.803381143Z Gathering data for ns/openshift-etcd...
+
+...
+
+Reprinting Cluster State:
+When opening a support case, bugzilla, or issue please include the following summary data along with any other requested information:
+ClusterID: 829er0fa-1ad8-4e59-a46e-2644921b7eb6
+ClusterVersion: Stable at "<cluster_version>"
+ClusterOperators:
+	All healthy and stable
 ----
 +
-<1> The default {product-title} `must-gather` image.
-<2> The `must-gather` image for low latency tuning diagnostics.
+When the command completes, the data is saved in a new directory that starts with `must-gather.local`. This directory is created in your current working directory.
 
 . Create a compressed file from the `must-gather` directory that was created in your working directory. For example, on a computer that uses a Linux operating system, run the following command:
 +
 [source,terminal]
 ----
- $ tar cvaf must-gather.tar.gz must-gather.local.5421342344627712289/ <1>
+$ tar cvaf must-gather.tar.gz must-gather-local.5421342344627712289// <1>
```

Comment:
In the build preview, the `/` is not displayed in the command, but the callout below includes the `/`. If you are attempting to print the `/`, you need a forward slash `\` to escape it: 
```
...-local.5421342344627712289/ \<1>
```

See: https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/#escape-with-backslashes